opentech/apply/determinations/tests/test_views.py

```diff
@@ -99,7 +99,7 @@ def test_can_edit_draft_determination(self):
         }, 'form')
         self.assertContains(response, '[Draft] Approved')
         self.assertContains(response, reverse(self.url_name.format('form'), kwargs=self.get_kwargs(submission)))
-        self.assertContains(response, 'Accepted determination draft message')
+        self.assertNotContains(response, 'Accepted determination draft message')
```

Comment:
@zerolab Your original code was creating an activity when the draft message was saving (see below) This shouldn't happen.
```
Sent a Needs more info determination for <title>: Accepted determination draft message
```